File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

/**
 * 品牌接口
 *
 * @author Administrator
 */
public interface BrandService {

    //查询所有
    public List<TbBrand> findAll();

    //分页
   // PageResult findPage(int pageNum, int pageSize);

    //添加品牌
    void add(TbBrand tbBrand);

    //数据辉县
    public TbBrand findOne(Long id);

    //修改
    void update(TbBrand tbBrand);

    //删除数据
    void delete(Long[] ids);


    //进行模糊查询
    PageResult findPage(TbBrand tbBrand,int pageNum, int pageSize);

    public List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService', function ($http) {
    //查询所有
    this.findAll = function () {
        return $http.get("../brand/findAll.do")
    };

    /* //分页查询
     this.findPage = function (page, size) {

         $http.get('../brand/findPage.do?page=' + page + '&size=' + size)
     };*/
    //数据回显
    this.findOne = function (id) {
        return  $http.get('../brand/findOne.do?id=' + id)

    };
    //添加数据
    this.add = function (entry) {

        return $http.post('../brand/add.do', entry)
    };
    //修改数据
    this.update = function (entry) {

        return $http.post('../brand/add.do', entry)
    };

    //批量删除
    this.dele = function (ids) {
        return $http.get('../brand/delete.do?ids=' + ids)

    };

    //模糊查询
    this.sea= function (page, size, searchEntry) {

        $http.post('../brand/search.do?page=' + page + '&size=' + size, searchEntry)
    }

    this.selectOptionList=function() {
        return $http.get('../brand/selectOptionList.do');
    }
});